integrationTests/vm/esdt/esdtProcess_test.go

```diff
@@ -80,22 +80,22 @@ func TestESDTIssueAndTransactionsOnMultiShardEnvironment(t *testing.T) {
 	nonce, round = integrationTests.WaitOperationToBeDone(t, nodes, nrRoundsToPropagateMultiShard, nonce, round, idxProposers)
 	time.Sleep(time.Second)
 
-	tokenIdenfitifer := string(getTokenIdentifier(nodes))
+	tokenIdentifier := string(getTokenIdentifier(nodes))
```

Comment:
also, please rename `initalSupply` to `initialSupply` inside `esdtProcess_test.go` file

Reply:
So many typos :)

Reply:
fixed